### PR TITLE
Add nunit/csharp support to learn-test

### PR DESCRIPTION
**Pull request by jdublu10**
This PR adds c# and nunit functionality to learn-test. Note that this will _not_ cause lights to light up– an endpoint needs to be added to ironboard for that to work.

### CODE REVIEW

File: lib/learn_test.rb [hunk 1]
```diff
@@ -31,8 +32,12 @@
 require_relative 'learn_test/strategies/karma'
 require_relative 'learn_test/strategies/protractor'
 require_relative 'learn_test/strategies/java_junit'
+
```
Comment (loganhasson) [inline lib/learn_test.rb]: super nitpicky of me, but would you mind removing these few extra newlines?

File: lib/learn_test/strategies/csharp_nunit.rb [hunk 2]
```diff
@@ -0,0 +1,55 @@
+require 'crack'
+require 'json'
+
+module LearnTest
+  module Strategies
+
```
Comment (loganhasson) [inline lib/learn_test/strategies/csharp_nunit.rb]: same thing here and after the class definition's `end`

**Comment by loganhasson** [comment]
@jdublu10 this is great...thanks! just left two really really overly-picky comments, but once you address those, we can merge away :)

**Comment by jdublu10** [comment]
Fixed those!

**Pull request merged**